syntax = "proto3";

package protocol;

message HeartbeatAck {
  uint64 received_at_ms = 1;
}
